### PR TITLE
Job details page

### PR DESCRIPTION
**Pull request by atanasovRazvan**
Completed job details.

### CODE REVIEW

File: src/job-details/JobDetails.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+import { ArrowDropDown } from '@material-ui/icons';
+import { Rating } from '@material-ui/lab';
+import { GET_JOB } from '../sources';
+import './styles.css';
+import CompanyDetails from './CompanyDetails';
+
+const JobDetails = () => {
+  const { id } = useParams();
+  const [companyCardVisibility, setCompanyCardVisibility] = useState(false);
+  const [job, setJob] = useState(null);
```
Comment (ih90) [inline src/job-details/JobDetails.js]: Yet again, any reason to have the job on the state ?

File: src/job-feed/JobCard.js [hunk 2]
```diff
@@ -1,73 +1,83 @@
...
-        <Typography
-          className="job-skill"
-          key={jobSkill.skill.id}
-          variant="body2"
-          component="p"
-        >
-          {jobSkill.skill.name}
+  id, name, description, companyName, jobSkills,
+}) => {
+  const [buttonClicked, setButtonClicked] = useState(false);
```
Comment (ih90) [inline src/job-feed/JobCard.js]: In a web app, we can have plenty of buttons. You might want to find a more suitable name for this.

File: src/job-details/JobDetailsPage.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import NavBar from '../home-page/NavBar';
+import JobDetails from './JobDetails';
+
+const JobDetailsPage = () => (
+  <div>
+    <NavBar />
```
Comment (ih90) [inline src/job-details/JobDetailsPage.js]: You should find a way to add the NavBar only once. If the app has N pages this is going to be ugly.

File: src/job-details/JobDetails.js (hunk 1 above)
Comment (atanasovRazvan) [reply src/job-details/JobDetails.js]: fixed

File: src/job-feed/JobCard.js (hunk 2 above)
Comment (atanasovRazvan) [reply src/job-feed/JobCard.js]: updated the name to 'jobDetailsButtonClicked'

File: src/job-details/JobDetailsPage.js (hunk 3 above)
Comment (atanasovRazvan) [reply src/job-details/JobDetailsPage.js]: updated, created a layout component

File: src/job-details/JobDetails.js [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+import { GET_JOB } from '../sources';
+import './styles.css';
+import CompanyDetails from './CompanyDetails';
+
+const JobDetails = () => {
+  const { id } = useParams();
+  const [companyCardVisibility, setCompanyCardVisibility] = useState(false);
+  const {
+    data, error,
+  } = useQuery(GET_JOB, { variables: { id: Number(id) } });
```
Comment (ih90) [inline src/job-details/JobDetails.js:19]: This works as expected, just something to keep in mind : `The ID type is serialized in the same way as a String; however, it is not intended to be human‐readable. While it is often numeric, it should always serialize as a String.`

**Review by ih90 — APPROVED** [review]

**Pull request merged**